### PR TITLE
Update readiness-assessment-fix.md

### PR DESCRIPTION
**Pull request by ElaineAnsell**
Changed the name of one of the checks, Security account names, to match the title of the check in ART-- which is called, Microsoft Managed Desktop service accounts. I also added wording to explain why we don't just tell them in this doc which account names they need to remedy, as listing them here would increase security risk for a malicious acting who can target that now known user account name.

@jaimeo , as always, I value your input on whether the wording I used is appropriate/easy to read. Thanks, Jaime!

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Comment by ElaineAnsell** [comment]
Adding @Jaime Ondrusek<mailto:jaimeo@microsoft.com> for review.

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 1]
```diff
@@ -277,13 +277,13 @@ A number of licenses are required to use Microsoft Managed Desktop.
 You don't have all the licenses you need to use Microsoft Managed Desktop. For more information, see [Microsoft Managed Desktop technologies](../intro/technologies.md) and [More about licenses](prerequisites.md#more-about-licenses).
 
 
-### Security account names
+### Microsoft Managed Desktop service accounts
 
-Certain security account names could conflict with ones created by Microsoft Managed Desktop.
+Certain accounts names could conflict with ones created by Microsoft Managed Desktop to manage the Microsoft Managed Desktop service.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
Certain account names could conflict with account names created by Microsoft Managed Desktop to manage the Microsoft Managed Desktop service.
```

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 2]
```diff
@@ -277,13 +277,13 @@ A number of licenses are required to use Microsoft Managed Desktop.
...
 
 **Not ready**
 
-You have at least one account name that will conflict with ones created by Microsoft Managed Desktop. Work with your Microsoft account representative to exclude these account names.
+You have at least one account name that will conflict with ones created by Microsoft Managed Desktop. Work with your Microsoft account representative to exclude these account names as we don't list the accounts names publicly to minimiize security risk.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
You have at least one account name that will conflict with account names created by Microsoft Managed Desktop. Work with your Microsoft account representative to exclude these account names as we don't list the account names publicly to minimize security risk. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**